Wow!
Thanks for the interest to this toy project!

I have tested your PR on my daily Intel i5-1155g7 laptop with

```
$ cat /etc/debian_version 
12.7

$ powerprofilesctl list
  performance:
    Driver:     intel_pstate
    Degraded:   no

  balanced:
    Driver:     intel_pstate

* power-saver:
    Driver:     intel_pstate
```

and I see that it works strange - the indicator shows many unneeded lines:

![indicator-with-PR](https://github.com/user-attachments/assets/7176faee-4fb5-4561-ad56-9a54a9cc61b6)

Probably you have to change the processing algorithm to something like 

```
powerprofilesctl list | grep ':$' | sed 's/:$//;s/\s*//g'
```

(from https://github.com/stonefeld/dotfiles/blob/148d9b86afaeefb74cfbbf34213ce8ce1cab8c4c/scripts/.local/bin/dmpower#L3 ) .

Also I have ran `pylint` 2.16.2 on the patched file. 
The most interesting warning here is the following:

```
powerprofilesctl-tray.py:96:17: W1510: 'subprocess.run' used without explicitly defining the value for 'check'. (subprocess-run-check)
```

Please update your PR with fixed code :)

--- a/powerprofilesctl-tray.py
+++ b/powerprofilesctl-tray.py
@@ -93,9 +93,13 @@ class Indicator:
     def get_available_profiles(self):
         """ Fetch available power profiles from powerprofilesctl """
 
-        result = subprocess.run(['powerprofilesctl', 'list'], stdout=subprocess.PIPE, text=True)
+        try:
+            result = subprocess.run(['powerprofilesctl', 'list'], stdout=subprocess.PIPE, text=True, check=True)
+        except subprocess.CalledProcessError as e:
+            print(f"An error occurred: {e}")
+
         lines = result.stdout.splitlines()
-        profiles = [line.replace('* ', '').replace(':', '').strip() for line in lines if ':' in line and 'PlatformDriver' not in line]
+        profiles = [line.replace('* ', '').replace(':', '').strip() for line in lines if line.endswith(':')]
         return profiles
 
     def change_performance_mode(self, source, string):

